elife/proofreader-php.sls

```diff
@@ -0,0 +1,6 @@
+proofreader-php:
+    cmd.run:
+        - name: composer global require elife/proofreader-php=dev-master
```

Comment:
They're probably fans of Postel: `Required package name optionally including a version constraint, e.g. foo/bar or foo/bar:1.0.0 or foo/bar=1.0.0 or "foo/bar 1.0.0"`